stylesheet.css

```diff
@@ -0,0 +1,187 @@
+body{
+	background-color: #f8f8f8;
+	margin: 0px;
+}
+#navbar{
```

Comment:
Can you please change all CSS selector to a class selector? Using a class selector make styling more reusable. I think it will be valuable to learn to do this earlier on with these projects.

Let's do this for all other occurrences.

Reply:
Thanks for your review. This has been improved on.